### PR TITLE
Mention the fact that authz plugins are available today

### PR DESCRIPTION
**Pull request by fntlnz**
I just added the authz plugins to the documentation page about plugins.

https://docs.docker.com/engine/extend/plugins/

cute-cat-pictures

Signed-off-by: Lorenzo Fontana fontanalorenzo@me.com

### CODE REVIEW

File: docs/extend/plugins.md [hunk 1]
```diff
@@ -65,6 +65,12 @@ Plugin
...
 [REX-Ray plugin](https://github.com/emccode/rexray)                                 | A volume plugin which is written in Go and provides advanced storage functionality for many platforms including VirtualBox, EC2, Google Compute Engine, OpenStack, and EMC.
 
+### Authorization plugins
+
+| Plugin                                                       | Description                                                                                                                                                                                           |
+|--------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Twistlock AuthZ Broker](https://github.com/twistlock/authz) | A basic extendable Docker authorization plugin that runs on directly on the host or inside a container. It basically allows you to define a few user policies that it evaluates during authorization. |
```
Comment (thaJeztah) [inline docs/extend/plugins.md]: Perhaps something like;

A basic extendable authorization plugin that runs directly on the host or inside a container. This plugin allows you to define user policies that it evaluates during authorization. Basic authorization is provided if Docker daemon is started with the `--tlsverify` flag (username is extracted from the certificate common name).

File: docs/extend/plugins.md [hunk 2]
```diff
@@ -65,6 +65,12 @@ Plugin
 [Quobyte Volume Plugin](https://github.com/quobyte/docker-volume)                   | A volume plugin that connects Docker to [Quobyte](http://www.quobyte.com/containers)'s data center file system, a general-purpose scalable and fault-tolerant storage platform.
 [REX-Ray plugin](https://github.com/emccode/rexray)                                 | A volume plugin which is written in Go and provides advanced storage functionality for many platforms including VirtualBox, EC2, Google Compute Engine, OpenStack, and EMC.
 
+### Authorization plugins
+
+| Plugin                                                       | Description                                                                                                                                                                                           |
```
Comment (thaJeztah) [inline docs/extend/plugins.md]: For consistency with the other tables in this document, can you remove the leading and trailing `|` in the table?

**Comment by fntlnz** [comment]
@thaJeztah Updated with your suggestions

File: docs/extend/plugins.md [hunk 3]
```diff
@@ -65,6 +65,11 @@ Plugin
 [Quobyte Volume Plugin](https://github.com/quobyte/docker-volume)                   | A volume plugin that connects Docker to [Quobyte](http://www.quobyte.com/containers)'s data center file system, a general-purpose scalable and fault-tolerant storage platform.
 [REX-Ray plugin](https://github.com/emccode/rexray)                                 | A volume plugin which is written in Go and provides advanced storage functionality for many platforms including VirtualBox, EC2, Google Compute Engine, OpenStack, and EMC.
 
+### Authorization plugins
+
+ Plugin                                                       | Description                                                                                                                                                                                                                            |
```
Comment (thaJeztah) [inline docs/extend/plugins.md]: oh, looks like you missed the trailing `|` here (sorry!)

Comment (fntlnz) [inline docs/extend/plugins.md]: :cat2: my fault

**Comment by thaJeztah** [comment]
LGTM, thanks!

**Comment by runcom** [comment]
LGTM

**Pull request merged**